astropy/cosmology/core.py

```diff
@@ -252,6 +252,11 @@ def __init__(self, H0, Om0, Ode0, Tcmb0=2.725, Neff=3.04,
         # Compute curvature density
         self._Ok0 = 1.0 - self._Om0 - self._Ode0 - self._Ogamma0 - self._Onu0
 
+        # Subclasses should override this reference if they provide
+        #  more efficient scalar versions of inv_efunc.
+        self._scalar_inv_efunc = self.inv_efunc
+        self._scalar_inv_efunc_args = ()
+
```

Comment:
Could you change this to `_inv_efunc_scalar` for consistency with `_tfunc_scalar` and `_xfunc_scalar`?